src/components/feedback/FeedbackList.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC } from 'react';
+import { useTheme } from '@emotion/react';
+
+import FeedbackItem from './FeedbackItem';
+import { feedbackListWrapperStyle, feedbackTotalCountStyle } from './style';
+
+import type { Feedback } from '@/@types/project';
+
+type FeedbackListProps = {
+  feedbacks: Feedback[];
+  totalFeedBackCount: number;
+  primary: boolean;
+};
+
+const FeedbackList: FC<FeedbackListProps> = ({
+  feedbacks,
+  primary,
+  totalFeedBackCount,
+}) => {
+  const theme = useTheme();
+  return (
+    <div>
+      <ul
+        css={feedbackListWrapperStyle}
+        style={feedbacks.length <= 2 ? { justifyContent: 'center' } : {}}
+      >
```

Comment:
`feedbacks.length <= 2`일 때 어떤 스타일을 적용할지에 대해서는 컴포넌트 부분이 아니라 스타일쪽에서 prop으로  처리하면 어떨까요?

```suggestion
      <ul
        css={feedbackListWrapperStyle(feedbacks.length <= 2)}
      >
```

Reply:
저도 css-in-js 사용하는 입장에선 이게 좋지 않을까 싶네요 !!!

Reply:
넵 반영하겠습니다 !